main.rb

```diff
@@ -1,15 +1,22 @@
 require_relative './app'
 require_relative './modules/choose_operation'
 require_relative './modules/display'
+require_relative './modules/load_data'
 
 @app = App.new
 
 def start_program
+  load_books(@app)
+  load_rentals(@app)
```

Comment:
- [ ] After creating rentals and exiting the app, When I run the app again I get get the below error. Please fix this issue and make sure that rentals are loaded successfully after starting the app.

![image](https://user-images.githubusercontent.com/83803414/200875823-135e4c73-ce55-4e11-ae31-bbe3b79ccc68.png)